projects/rero/ng-core/src/lib/record/search/aggregation/aggregation.component.html

```diff
@@ -24,11 +25,18 @@ <h5 class="mb-0">
   </div>
   <div class="collapse" [class.show]="showAggregation()">
     <div class="card-body">
-      <ng-core-record-search-aggregation-buckets
-      [buckets]="aggregation.value.buckets"
-      [aggregationKey]="aggregation.key"
-      [size]="aggregation.bucketSize"
-      ></ng-core-record-search-aggregation-buckets>
+      <!-- Checkbox -->
+      <ng-container *ngIf="aggregation.type === 'checkbox'">
+        <ng-core-record-search-aggregation-buckets [buckets]="aggregation.value.buckets"
+          [aggregationKey]="aggregation.key" [size]="aggregation.bucketSize">
+        </ng-core-record-search-aggregation-buckets>
+      </ng-container>
+
+      <!-- Slider -->
+      <ng-container *ngIf="aggregation.type === 'slider'">
```

Comment:
Can we use ES terminology for type? https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations.html